### ISSUE TITLE
[Bug] [deploy] helm install error, no template "dolphinscheduler.zookeeper.env_vars" associated with template "gotpl" 

### ISSUE THREAD

**Issue by HeChuanXUPT** [apache/dolphinscheduler#7438]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.

### What happened

Helm install dolphinscheduler failed.when deploy with k8s.
https://dolphinscheduler.apache.org/zh-cn/docs/latest/user_doc/guide/installation/kubernetes.html

### What you expected to happen

```
# helm dependency update .
Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "bitnami" chart repository
Update Complete. ⎈Happy Helming!⎈
Saving 2 charts
Downloading postgresql from repo https://charts.bitnami.com/bitnami
Downloading zookeeper from repo https://charts.bitnami.com/bitnami
Deleting outdated charts

# helm install dolphinscheduler .
Error: INSTALLATION FAILED: template: dolphinscheduler/templates/statefulset-dolphinscheduler-worker.yaml:72:16: executing "dolphinscheduler/templates/statefulset-dolphinscheduler-worker.yaml" at <include "dolphinscheduler.zookeeper.env_vars" .>: error calling include: template: no template "dolphinscheduler.zookeeper.env_vars" associated with template "gotpl"
```

### How to reproduce

```
$ tar -zxvf apache-dolphinscheduler-2.0.0-src.tar.gz
$ cd apache-dolphinscheduler-2.0.0-src/docker/kubernetes/dolphinscheduler
$ helm repo add bitnami https://charts.bitnami.com/bitnami
$ helm dependency update .
$ helm install dolphinscheduler .
```

### Anything else

helm version: 3.7.2
Kubernetes version: 1.18

### Version

2.0.0

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by kezhenxu94** [comment]
This should be fixed and will be released in 2.0.2

**Comment by lenboo** [comment]
Fixed in 2.0.2 and close it.
